package.json

```diff
@@ -29,6 +29,7 @@
     "bignumber.js": "^9.0.1",
     "ethers": "^5.0.24",
     "honeyswap-default-token-list": "^3.1.1",
+    "lodash.debounce": "^4.0.8",
```

Comment:
I thought we were removing this... I guess this is needed to remove it?

Reply:
Its a small subset of the big lodash library just giving us the debounce function which I wanna use to debounce the validation and parsing of the csv.